pkgs/tools/system/thermald/default.nix

```diff
@@ -36,7 +58,8 @@ stdenv.mkDerivation rec {
   meta = with stdenv.lib; {
     description = "Thermal Daemon";
     homepage = "https://01.org/linux-thermal-daemon";
-    license = licenses.gpl2;
+    changelog = "https://github.com/intel/thermal_daemon/blob/master/README.txt";
+    license = licenses.gpl2Only;
```

Comment:
From the changelog:

```
Version 0.5
- License update to GPL v2 or later
```

Reply:
Ah, I looked at the license, but did not check any of the source files. Fixed.